stanford_news.strongarm.inc

```diff
@@ -59,20 +59,26 @@ function stanford_news_strongarm() {
       '300_flexwidth' => array(
         'custom_settings' => FALSE,
       ),
+      'stanford_news_extras' => array(
```

Comment:
I believe this is just a listing of the view modes available. It's in the view_modes array.